docs/encyclopedia/retry-policies.mdx

```diff
@@ -119,6 +119,12 @@ The wait time before a retry is the _retry interval_. A retry interval is the sm
 
 <PrettyImage src="/img/retry-interval-diagram.png" title="Diagram that shows the retry interval and its formula" />
 
+### Per-error next retry delay
+
+Sometimes, your Activity or Workflow raises a special exception that needs a different retry interval from the Retry Policy.
+To accomplish this, you may throw an [Application Failure](/references/failures#application-failure) with the next retry delay field set. This value will replace and override whatever the retry interval would be on the retry policy.
```

Comment:
```suggestion
To accomplish this, you may throw an [Application Failure](/references/failures#application-failure) with the Next Retry Delay field set. This value will replace and override whatever the retry interval would be on the Retry Policy.
```

Unclear what Next Retry Delay field is - is that a per-SDK field? Should it be capitalized?

Reply:
Should this link right to the Java section that explains this?